addons/account/static/src/components/x2many_buttons/x2many_buttons.js

```diff
@@ -0,0 +1,61 @@
+/** @odoo-module **/
+
+import { standardFieldProps } from "@web/views/fields/standard_field_props";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Component } from "@odoo/owl";
+
+class X2ManyButtons extends Component {
+    static template = "account.X2ManyButtons";
+    static props = standardFieldProps;
+
+    setup() {
+        this.action = useService("action");
+        this.orm = useService("orm");
+    }
+
+    async openTreeAndDiscard() {
+        const ids = this.currentField.currentIds;
+        await this.props.record.discard();
```

Comment:
I emplemented a method that checks if it;s in edition before calling `this.props.record.discard();`, but i figured there is no need for that since discard already does that